src/plugins/visualizations/public/types.ts

```diff
@@ -35,7 +37,7 @@ export interface VisualizationController {
 
 export type VisualizationControllerConstructor = new (
   el: HTMLElement,
-  vis: Vis
+  vis: ExprVis
```

Comment:
:information_source: It seems there is currently two very similar Vis classes around `ExprVis` and `Vis` and they've got mixed up in a couple of places. `ExprVis` is the instance used in the renderer when intatiating the visualizations, while `Vis` is the one used by the editor instantiated at a very high level.